birdhouse/default.env

```diff
@@ -11,6 +11,9 @@ export THREDDS_IMAGE="unidata/thredds-docker:4.6.15"
 # Folder on the host to persist Jupyter user data (noteboooks, HOME settings)
 export JUPYTERHUB_USER_DATA_DIR="/data/jupyterhub_user_data"
 
+# Activates mounting a tutorial-notebooks subfolder that has the same name as the spawned image on JupyterHub
+export MOUNT_IMAGE_SPECIFIC_NOTEBOOKS=false
```

Comment:
In which case would we set this to false?

Reply:
I simply added this option to have an easy way to activate or deactivate the mounting of a folder of the same name as the spawned image name.
I think in our case, for CRIM, we will always set this to True, but for other organizations, the option will be there, if they don't want to use this feature.

Reply:
Backcompatibility I guess? Could be an option until all images take advantage of the new mecanism.

Reply:
@ChaamC In this case, maybe we should have this option defined only in the `env.local` and not `default.env`, since it's a preference. 
Having this `MOUNT_IMAGE_SPECIFIC_NOTEBOOKS` variable defined in both files will lead to confusion.